services/spar/src/Spar/API.hs

```diff
@@ -517,9 +514,9 @@ idpCreateXML ::
   SAML.IdPMetadata ->
   Maybe SAML.IdPId ->
   Maybe WireIdPAPIVersion ->
-  Maybe (Range 1 32 Text {- human-readable idp name -}) ->
+  Maybe (Range 1 32 Text) ->
   Sem r IdP
-idpCreateXML zusr rawIdpMetadata idpmeta mReplaces (fromMaybe defWireIdPAPIVersion -> apiversion) mHandle = withDebugLog "idpCreateXML" (Just . show . (^. SAML.idpId)) $ do
+idpCreateXML zusr rawIdpMetadata idpmeta mReplaces (fromMaybe defWireIdPAPIVersion -> apiversion) mHandle {- unique human-readable idp name -} = withDebugLog "idpCreateXML" (Just . show . (^. SAML.idpId)) $ do
```

Comment:
This comment should be added to the data structure instead, that is, in https://github.com/wireapp/wire-server/blob/f1c032a3d18717f519912825f605707f400b9b4f/libs/wire-api/src/Wire/API/User/IdentityProvider.hs#L52.